src/library_html5.js

```diff
@@ -194,7 +194,7 @@ var LibraryHTML5 = {
         {{{ makeSetValue('varargs', 0, 'eventTypeId', 'i32') }}};
         {{{ makeSetValue('varargs', 4, 'eventData', 'i32') }}};
         {{{ makeSetValue('varargs', 8, 'userData', 'i32') }}};
-        __emscripten_call_on_thread(0, targetThread, {{{ cDefine('EM_FUNC_SIG_IIII') }}}, eventHandlerFunc, eventData, varargs);
+        _emscripten_dispatch_to_thread_(targetThread, {{{ cDefine('EM_FUNC_SIG_IIII') }}}, eventHandlerFunc, eventData, varargs);
```

Comment:
Whats with the underscore suffix?  That is not a pattern we normally use in emscripten.   If you want to signify that the function takes varags perhaps put `v` in it somewhere a. la `vprintf`?   Or maybe `_emscripten_dispatch_to_thread_va`?

Reply:
The extra underscore is to differentiate this from the macro `emscripten_dispatch_to_thread` that wraps the function and provides a cast of the function pointer to `void*` so that C++ users do not have to do that themselves. This pattern is also used for `emscripten_{sync,async,async_waitable}_run_in_main_runtime_thread`.